client/src/index.css

```diff
@@ -1,7 +1,21 @@
+@import url("./styles/_base.css");
+@import url("./styles/_colors.css");
+@import url("./styles/_fonts.css");
+
+@font-face {
+  font-family: "HammersmithOne";
+  src: local("HammersmithOne"),
+    url("./assets/Font/HammersmithOne-Regular.ttf") format("truetype");
+}
+
 *,
 ::before,
 ::after {
   box-sizing: content-box;
```

Comment:
`content-box` is default, you probably want `border-box` which is the better way. 
https://css-tricks.com/box-sizing/